referenceframe/frame.go

```diff
@@ -490,14 +459,18 @@ func (mf *mobile2DFrame) Transform(input []Input) (spatial.Pose, error) {
 	if err != nil && !strings.Contains(err.Error(), OOBErrString) {
 		return nil, err
 	}
-	return spatial.NewPoseFromPoint(r3.Vector{input[0].Value, input[1].Value, 0}), err
+	return spatial.NewPose(
+		r3.Vector{X: input[0].Value, Y: input[1].Value, Z: 0},
+		&spatialmath.OrientationVector{OZ: 1, Theta: input[2].Value},
+	), err
 }
 
 // InputFromProtobuf converts pb.JointPosition to inputs.
 func (mf *mobile2DFrame) InputFromProtobuf(jp *pb.JointPositions) []Input {
 	n := make([]Input, len(jp.Values))
 	for idx, d := range jp.Values {
 		n[idx] = Input{d}
+		// TODO: should this convert deg to rad?
```

Comment:
This should convert deg to rad for theta, yes.

Reply:
This structure doesn't exist anymore, going to resolve this